[2023-03-20 19:01:47,060] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.9.create_postgres_table_precipitation manual__2023-03-20T19:01:39.369448+00:00 [queued]>
[2023-03-20 19:01:47,096] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: to_posgrest_v4.9.create_postgres_table_precipitation manual__2023-03-20T19:01:39.369448+00:00 [queued]>
[2023-03-20 19:01:47,097] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 19:01:47,098] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-03-20 19:01:47,099] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-20 19:01:47,137] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_postgres_table_precipitation> on 2023-03-20 19:01:39.369448+00:00
[2023-03-20 19:01:47,144] {standard_task_runner.py:52} INFO - Started process 22983 to run task
[2023-03-20 19:01:47,151] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'to_posgrest_v4.9', 'create_postgres_table_precipitation', 'manual__2023-03-20T19:01:39.369448+00:00', '--job-id', '233', '--raw', '--subdir', 'DAGS_FOLDER/to_postgresql.py', '--cfg-path', '/tmp/tmprxt71rai', '--error-file', '/tmp/tmptu4lalj6']
[2023-03-20 19:01:47,153] {standard_task_runner.py:77} INFO - Job 233: Subtask create_postgres_table_precipitation
[2023-03-20 19:01:47,452] {logging_mixin.py:109} INFO - Running <TaskInstance: to_posgrest_v4.9.create_postgres_table_precipitation manual__2023-03-20T19:01:39.369448+00:00 [running]> on host 0b4cdb459521
[2023-03-20 19:01:47,765] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=sakabuana31
AIRFLOW_CTX_DAG_ID=to_posgrest_v4.9
AIRFLOW_CTX_TASK_ID=create_postgres_table_precipitation
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T19:01:39.369448+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T19:01:39.369448+00:00
[2023-03-20 19:01:47,843] {base.py:79} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: final, Login: ***, Password: ***, extra: {}
[2023-03-20 19:01:47,916] {dbapi.py:225} INFO - Running statement: 
            create table if not exists precipitation (
                date DATE PRIMARY KEY,
                precipitation DECIMAL(5, 2),
                precipitation_normal DECIMAL(5, 2),
                status_preci int
            )
        , parameters: None
[2023-03-20 19:01:47,920] {postgres.py:71} INFO - NOTICE:  relation "precipitation" already exists, skipping

[2023-03-20 19:01:48,009] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=to_posgrest_v4.9, task_id=create_postgres_table_precipitation, execution_date=20230320T190139, start_date=20230320T190147, end_date=20230320T190148
[2023-03-20 19:01:48,093] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-20 19:01:48,339] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
